pdh_check/datadog_checks/pdh_check/data/conf.yaml.example

```diff
@@ -55,6 +55,13 @@ instances:
     #   - active_directory.dra.inbound.bytes.total,int
     #   - active_directory.ldap.bind_time,float
 
+    ## @param admin_share - string - optional - default: c$
```

Comment:
Can we move this whole block next to `host/username/password` ?

Reply:
ok